tests/app/templating/test_template_renderer.py

```diff
@@ -62,4 +59,35 @@ def test_household_summary_values_are_escaped(self):
         }
 
         rendered = TemplateRenderer().render(description, **context)
-        self.assertEqual(rendered, '<h2 class=\'neptune\'>Your household includes:</h2> <ul><li>Alice Aardvark</li><li>Bob Berty Brown</li><li>\ " !</li><li>Dave Dixon Davies</li></ul>')
+        self.assertEqual(rendered,
+                         '<h2 class=\'neptune\'>Your household includes:</h2> <ul><li>Alice Aardvark</li><li>Bob Berty Brown</li><li>\ " !</li><li>Dave Dixon Davies</li></ul>')
+
+    def test_render_nested_templatable_property(self):
+        question = Question()
+        question.guidance = [
+            {
+                'title': 'Include',
```

Comment:
As discussed, this is the only thing I wasn't sure about, but from talking it sounds like we can still pipe into the title. Otherwise looks good.